src/style.css

```diff
@@ -1 +1,140 @@
-/* empty source */
+/* #region page */
+
+* {
+  margin: 0;
+  padding: 0;
+}

```

Comment:
Set only `box-sizing: border-box` for `*` selector. Properties such margin and padding could be set for selectors which really need them